src/test/scala/edu/berkeley/ce/rockslicing/BlockVTKSpec.scala

```diff
@@ -0,0 +1,41 @@
+package edu.berkeley.ce.rockslicing
+
+import org.scalatest._
+import scala.math.sqrt
+
+class BlockVTKSpec extends FunSuite {
+  val boundingFaces = List(
+    Face((-1.0, 0.0, 0.0), 0.0, phi=0, cohesion=0), // -x = 0
+    Face((1.0, 0.0, 0.0), 1.0, phi=0, cohesion=0),  // x = 1
+    Face((0.0, -1.0, 0.0), 0.0, phi=0, cohesion=0), // -y = 0
+    Face((0.0, 1.0, 0.0), 1.0, phi=0, cohesion=0),  // y = 1
+    Face((0.0, 0.0, -1.0), 0.0, phi=0, cohesion=0), // -z = 0
+    Face((0.0, 0.0, 1.0), 1.0, phi=0, cohesion=0)   // z = 1
+  )
+  val unitCube = Block((0.0, 0.0, 0.0), boundingFaces)
+
+  val slantedJoint = Joint((1.0/sqrt(2.0), 1.0/sqrt(2.0), 0.0), (0.0, 0.0, 0.0),
+                           (1.0, 0.0, 0.0), 0.0, 0.0, shape = Nil)
+
+  test("Getting to the bottom of this bullshit") {
```

Comment:
I think this test case should have a more informative name. Not that the current one isn't interesting. :)
Also, what's with all the comments? Maybe left over from debugging?